### PR TITLE
[104] Todos for 104

### PR DESCRIPTION
**Pull request by michdud**

### CODE REVIEW

File: mdc_100_series/lib/supplemental/product_columns.dart [hunk 1]
```diff
@@ -33,8 +33,10 @@ class TwoProductCardColumn extends StatelessWidget {
 
       double heightOfCards = (constraints.biggest.height - spacerHeight) / 2.0;
       double heightOfImages = heightOfCards - ProductCard.kTextBoxHeight;
+      // TODO: Change imageAspectRatio calculation (104)
       double imageAspectRatio = constraints.biggest.width / heightOfImages;
 
+      //TODO: Replace Column with a ListView (104)
```
Comment (willlarche) [inline mdc_100_series/lib/supplemental/product_columns.dart]: Space after slash

File: mdc_100_series/lib/app.dart [hunk 2]
```diff
@@ -13,18 +13,27 @@
...
+// TODO: Convert ShrineApp to stateful widget(104)
 class ShrineApp extends StatelessWidget {
+  // TODO: Add variable for selected Category (104)
+  // TODO: Add a callback when a Category is tapped (104)
+
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
       title: 'Shrine',
+      // TODO: Implement home with a Backdrop (104)
```
Comment (willlarche) [inline mdc_100_series/lib/app.dart]: This one could be clearer.

File: mdc_100_series/lib/app.dart [hunk 3]
```diff
@@ -13,18 +13,27 @@
...
       home: HomePage(),
+      // TODO: Pass _currentCategory for frontLayer (104)
+      // TODO: Change backLayer field to CategoryMenuPage (104)
```
Comment (willlarche) [inline mdc_100_series/lib/app.dart]: add 'value' after 'field'

**Review by willlarche — CHANGES_REQUESTED** [review]
Thank you!

File: mdc_100_series/lib/app.dart (hunk 2 above)
Comment (michdud) [reply mdc_100_series/lib/app.dart]: Done!

File: mdc_100_series/lib/app.dart (hunk 3 above)
Comment (michdud) [reply mdc_100_series/lib/app.dart]: Done!

File: mdc_100_series/lib/supplemental/product_columns.dart (hunk 1 above)
Comment (michdud) [reply mdc_100_series/lib/supplemental/product_columns.dart]: Done!

**Comment by michdud** [comment]
PTAL!

**Review by willlarche — APPROVED** [review]
Thank you!

**Pull request merged**